Validation/MuonME0Digis/BuildFile.xml

```diff
@@ -0,0 +1,28 @@
+<use name="boost"/>
+<use name="DataFormats/MuonDetId"/>
+<use name="DataFormats/CLHEP"/>
+<use name="DataFormats/GEMDigi"/>
+<use name="FWCore/Framework"/>
+<use name="FWCore/MessageLogger"/>
+<use name="FWCore/ParameterSet"/>
+<use name="FWCore/Utilities"/>
+<use name="MagneticField/Engine"/>
+<use name="MagneticField/Records"/>
+<use name="TrackingTools/GeomPropagators"/>
+<use name="TrackingTools/Records"/>
+<use name="TrackingTools/TrajectoryState"/>
+<use name="Geometry/CommonDetUnit"/>
+<use name="Geometry/Records"/>
+<use name="Geometry/GEMGeometry"/>
+<use name="Geometry/CommonTopologies"/>
+<use name="SimDataFormats/TrackingHit"/>
+<use name="SimDataFormats/Track"/>
+<use name="SimDataFormats/Vertex"/>
+<use name="CommonTools/UtilAlgos"/>
+<use name="Utilities/General"/>
+<use name="DQMServices/Core"/>
+<use name="rootcore"/>
+<use name="Validation/MuonME0Hits"/>
```

Comment:
the build files should just have the packages that you include in cc or h files. Most of these should be removed.